### PR TITLE
Plugfest page reorganization

### PR DESCRIPTION
**Pull request by egekorkan**
1. I have moved the content
2. Removed HTML headers that come from Hugo anyway
3. Named it to services since plugfest is a W3C specific term
4. Made the GitHub link only a logo since the menu items were getting split -> We need to think of the header menu

I did not change the content much. Once it is deployed, I think we should add a redirect to plugfest.thingweb.io

### CODE REVIEW

File: content/services.html [hunk 1]
```diff
@@ -1,151 +1,11 @@
...
-
-    <h1>Eclipse Thingweb Services for W3C WoT</h1>
+---
+title: "Services"
+date: 2023-10-20T09:35:33+02:00
+draft: false
+menu: "main"
+---
+    
+    <h1>Services W3C WoT Developers</h1>
```
Comment (JKRhb) [inline content/services.html:8]:
```suggestion
    <h1>Services for W3C WoT Developers</h1>
```

File: content/services.html [hunk 2]
```diff
@@ -258,39 +118,4 @@ <h2>Online Things for Testing</h2>
...
-      </ul>
-    </div>
-    <div>
-      <p>Copyrights © 2023 Eclipse Thingweb. All Rights Reserved.</p>
-    </div>
-  </footer>
-</body>
-
-</html>
+  </div>
```
Comment (JKRhb) [inline content/services.html:121]:
```suggestion
  </div>

```

**Review by JKRhb — COMMENTED** [review]
Looks go to me! :) The link on the home page (the one to "Online Things") would probably also need to be adjusted, right?

Also spotted two tiny nits down below:

**Comment by egekorkan** [comment]
@JKRhb I have fixed one link to online things but it is in other places as well (GitHub landing, Eclipse Page) so I will do that once the PR is merged

**Pull request merged**